assets/dev/js/editor/controls/color.js

```diff
@@ -62,6 +65,19 @@ export default class extends ControlBaseDataView {
 		jQuery( this.colorPicker.picker.getRoot().root ).addClass( 'elementor-control-unit-1 elementor-control-tag-area' );
 	}
 
+	hidePickerOnPreviewClick() {
+		const picker = this.colorPicker.picker;
+		const pickerUtils = picker.constructor.utils;
+
+		picker._eventBindings.push(
+			pickerUtils.on( elementorFrontend.elements.window.document, [ 'touchstart', 'mousedown' ], () => {
```

Comment:
Maybe it will be better to use `pointerdown`
https://developer.mozilla.org/en-US/docs/Web/API/Element/pointerdown_event